en/admin/config.rst

```diff
@@ -1601,6 +1611,64 @@ The following are parameters related to SQL statements and data types supported
       
         1, NULL
 
+.. _thread-parameters:
+
+Thread-Related Parameters
+-------------------------
+
+Thread managemenet can be confirued by threads parameters. The type and value range for each parameter are as follows:
+
++---------------------------------------+--------+-------------------+----------+----------+
+| Parameter Name                        | Type   | Default           | Min      | Max      |
++=======================================+========+===================+==========+==========+
+| thread_connection_pooling             | bool   | true              |          |          |
++---------------------------------------+--------+-------------------+----------+----------+
+| thread_connection_timeout_seconds     | int    | 300               | -1       | 3600     |
++---------------------------------------+--------+-------------------+----------+----------+
+| thread_worker_pooling                 | bool   | true              |          |          |
++---------------------------------------+--------+-------------------+----------+----------+
+| thread_worker_timeout_seconds         | int    | 300               | -1       | 3600     |
++---------------------------------------+--------+-------------------+----------+----------+
+| loaddb_worker_count                   | bool   | 8                 | 2        | 64       |
++---------------------------------------+--------+-------------------+----------+----------+
+
+**thread_connection_pooling**
+
+    If **thread_connection_pooling** parameter is true, all threads used for client connection management are pooled on server boot.
+
+**thread_connection_timeout_seconds**
+
+    **thread_connection_timeout_seconds** is a parameter that configures \
+    \wait time before stopping for threads handling connection management. \
+    \After closing a connection, the thread will wait the value of the \
+    \parameter expressed in seconds to be assigned a new connection. If no \
+    \connection is assigned and the wait time expires, the thread stops. \
+    \Another thread may be started the next time a connection comes. \
+    \If parameter value is **-1**, threads never stop. They sleep until \
+    \they are given a new assignment.
+
+**thread_worker_pooling**
+
+    If **thread_connection_pooling** parameter is true, all threads used for client requests execution are pooled on server boot.
+
+**thread_worker_timeout_seconds**
+
+    **thread_worker_timeout_seconds** is a parameter that configures \
+    \wait time before stopping for threads handling client requests. \
+    \After executing a request, the thread will wait the value of the \
```

Comment:
Now, the worker threads only handle client requests work, but in we may push other work which is not originated by client. What about reformulating to give a more generic meaning ?

Reply:
I am afraid "work" would be too generic and thus without real meaning.

Reply:
I support "client request:".

Reply:
Ok, let's keep client request for now. If in the future pool is changed to do something else, we can think of better naming.